DSSE/environment/coverage_env.py

```diff
@@ -120,5 +159,17 @@ def step(self, actions: dict[str, int]) -> tuple:
             self.agents = []
         return observations, rewards, terminations, truncations, infos
 
+    def compute_infos(self, is_completed: bool) -> dict[str, dict]:
+        # TODO: Is this the best way to inform the coverage rate, Cum_pos and repetitions?
+        total_states = len(self.seen_states) + len(self.not_seen_states)
```

Comment:
Why not `total_states = len(self.all_states)` or even `total_states = self.grid_size ** 2`?
This is not a complaint, just thought it could be simpler that way.

Reply:
You have to remove the cells with 0 probability from the not_seen_states after the pre-simulation, so the total len isnt len(all_states) anymore.

Reply:
Got it!